src/app/src/main/java/com/example/bookmark/AcceptedBookDetailsActivity.java

```diff
@@ -90,23 +91,17 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == AcceptedBookDetailsActivity.GET_ISBN && resultCode == Activity.RESULT_OK) {
             String isbn = data.getStringExtra("ISBN");
             if(book.getIsbn().equals(isbn)) {
-                StorageServiceProvider.getStorageService().retrieveRequest(
+                Request request = RequestUtil.retrieveRequestsOnBookByStatus(book, Request.Status.ACCEPTED, this);
+                book.setStatus(Book.Status.BORROWED);
+                request.setStatus(Request.Status.BORROWED);
+                StorageServiceProvider.getStorageService().storeRequest(
+                    request,
+                    aVoid -> Log.d("Accepted Book Details", "Request stored"),
```

Comment:
Could define `private static final String TAG = "Accepted Book Details";` and use `TAG` here instead.